.github/scripts/setup.sh

```diff
@@ -8,10 +8,10 @@ echo "Running .github/scripts/setup.sh..."
 conda init
 conda info --envs
 
-wget -O 2021_09_02_gtsam_python38_wheel.zip --no-check-certificate "https://drive.google.com/uc?export=download&id=1GSf3NxabSCsu2nPcBUuEhhVq_m1FvdY5"
+wget https://github.com/johnwlambert/gtsfm-datasets-mirror/releases/download/v4.2a5/gtsam-4.2a5-cp38-cp38-manylinux2014_x86_64.whl.zip
 
-unzip 2021_09_02_gtsam_python38_wheel.zip
-pip install gtsam-4.1.0-cp38-cp38-manylinux2014_x86_64.whl
+unzip gtsam-4.2a5-cp38-cp38-manylinux2014_x86_64.whl.zip
+pip install gtsam-4.2a5-cp38-cp38-manylinux2014_x86_64.whl
```

Comment:
nit: you can export the name to make it easier from next time.

Reply:
udpated, hopefully the change is what you meant.